tools/modules/system/module_armbian_firmware.sh

```
@@ -233,16 +236,16 @@ function module_armbian_firmware() {
 				local test_hold=($(for all_packages in ${packages[@]}; do
 					for hold_packages in ${get_hold[@]}; do
-					echo $all_packages | grep $hold_packages
+						echo $all_packages | grep $hold_packages
 					done
 				done))
-			[[ -z ${test_hold[@]} ]] && return 1 || return 0
+				[[ -z ${test_hold[@]} ]] && return 1 || return 0
 			else
 				# put Armbian packages on hold
-				apt-mark hold ${packages[@]} >/dev/null 2>&1
+				apt-mark hold ${packages[@]} > /dev/null 2>&1
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)